### PR TITLE
added my notes and links to foursquare

### PR DESCRIPTION
**Pull request by brunoamaral**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,23 +9,48 @@ This page is a tourist guide to the beautiful city of Lisbon. It's on GitHub bec
 
 * [The Nations Park Gondola Lift][3] (Cable Car)
 * [São Jorge Castle][4]
-* [Elevador da Bica)[5]
+* [Elevador da Bica][5]
```
Comment (dramalho) [inline README.md]: why not link it straight away? Footnotes are lovely and all, but in this context?

**Comment by dramalho** [comment]
:+1:

**Comment by brunoamaral** [comment]
@dramalho i agree, blame @celso ! 

I just followed the same format that was already on the file.

**Comment by dramalho** [comment]
I need to see the coding standard please, I'm not sure this provides any benefit to the stakeholders / shareholders and assorted persons holding things

**Pull request merged**